helix-core/src/main/java/org/apache/helix/HelixManagerFactory.java

```diff
@@ -61,6 +61,22 @@ public static HelixManager getZKHelixManager(String clusterName, String instance
     return new ZKHelixManager(clusterName, instanceName, type, zkAddr, stateListener);
   }
 
+  /**
```

Comment:
nit:  this comment is same as all other constructors. 
What is the unique difference of this constructor needs to be documented?